packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.vue

```diff
@@ -37,7 +35,7 @@ const requiredMarkerClass = computed(() => {
     <OnyxSkeleton v-if="!props.hideLabel" class="onyx-checkbox-skeleton__label" />
   </div>
 
-  <label v-else class="onyx-checkbox" :class="[requiredMarkerClass]">
+  <label v-else class="onyx-checkbox" :class="[requiredMarkerClass, requiredTypeClass]">
```

Comment:
Didn't jannick say the marker type should only be set on application level?

Reply:
Yes. This property allows overriding the application default per component.